### PR TITLE
feat: add Can I Use extension

### PR DESCRIPTION
**Pull request by thomaslombart**
## Description

Add "Can I Use" extension allowing you to get up-to-date browser support for front-end web technologies on desktop and mobile web browsers.

It's also possible to add a project configured with Browserlist to get supported technologies from the main view.

## Type of change

- New extension

## Screencast

https://user-images.githubusercontent.com/16003285/146597379-2fb1805e-6253-4a74-8085-16453a1494d0.mp4

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)

### CODE REVIEW

**Comment by zepfietje** [comment]
Damn this looks really neat! Great work @thomaslombart, can't wait to use this extension! 🚀

File: extensions/can-i-use/src/index.tsx [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+        return (
+          <List.Item
+            key={featureName}
+            title={feat.title}
+            keywords={[featureName]}
+            accessoryTitle={statusToName[feat.status]}
+            {...(accessoryIcon ? { accessoryIcon } : {})}
+            actions={
+              <ActionPanel>
+                <PushAction title="See details" target={<FeatureDetail feature={featureName} />} />
```
Comment (fe9lix) [inline extensions/can-i-use/src/index.tsx:48]:
```suggestion
                <PushAction title="Show Details" target={<FeatureDetail feature={featureName} />} />
```

Comment (fe9lix) [reply extensions/can-i-use/src/index.tsx:48]: Please also specify a suitable icon here so that all actions in the panel show an icon.

**Review by fe9lix — CHANGES_REQUESTED** [review]
Hey @thomaslombart – thanks for your contribution. I've left a comment regarding the primary action (icon, label) but the main issue is that there's a bug when opening the details. Here's the error: 

```
TypeError: Cannot read properties of undefined (reading 'getSupport')

  const supportTable = caniuse.getSupport(feature);
                               ^

FeatureDetail:FeatureDetail.tsx:11:32

---
 8: }
 9: 
10: export default function FeatureDetail({ feature }: FeatureDetailProps) {
11:   const supportTable = caniuse.getSupport(feature);
12: 
13:   return (
14:     <List searchBarPlaceholder="Search browsers...">
---

renderWithHooks:react-reconciler.development.js:5636:22
mountIndeterminateComponent:react-reconciler.development.js:7756:17
beginWork:react-reconciler.development.js:8543:18
Object.invokeGuardedCallbackProd:react-reconciler.development.js:9534:14
invokeGuardedCallback:react-reconciler.development.js:9621:35
beginWork$1:react-reconciler.development.js:12459:11
performUnitOfWork:react-reconciler.development.js:11682:16
workLoopSync:react-reconciler.development.js:11636:9
renderRootSync:react-reconciler.development.js:11611:11
```

**Comment by thomaslombart** [comment]
@fe9lix Would you mind sharing how you got the error? I can't reproduce it locally. 
What feature have you chosen? Did you enter some parameters in the extension?

**Comment by fe9lix** [comment]
> @fe9lix Would you mind sharing how you got the error? I can't reproduce it locally.
> What feature have you chosen? Did you enter some parameters in the extension?

So you can repro if you bump up the `@raycast/api` dependency to `^1.26.3`, then `npm install` again and `npm run dev`.
The reason is that the dev tool of this version produces a "more correct" build for module exports. 
The fix in your case seems to be to change the two `caniuse` imports to the following:
`import * as caniuse from "caniuse-api";`

**Comment by thomaslombart** [comment]
I was able to reproduce the bug on 1.26.3. It's a subtle one indeed! It should be good now.

**Review by fe9lix — APPROVED** [review]

**Pull request merged**